ingestion/src/metadata/ingestion/source/database/datalake.py

```diff
@@ -256,6 +256,14 @@ def get_tables_name_and_type(  # pylint: disable=too-many-branches
                 bucket = self.client.get_bucket(bucket_name)
                 for key in bucket.list_blobs(prefix=prefix):
                     table_name = self.standardize_table_name(bucket_name, key.name)
+                    # adding this condition as the gcp blobs also contains directory, which we can filter out
+                    if table_name.endswith("/") or not self.check_valid_file_type(
+                        key.name
+                    ):
+                        logger.debug(
+                            f"Object filtered due to unsupported file type: {key.name}"
+                        )
+                        continue
```

Comment:
this should reduce ingestion failures and es calls for unsupported types and directories